scripts/test-functions-deploy.js

```diff
@@ -329,10 +312,6 @@ var main = function () {
       console.log(clc.green("\u2713 Test passed: creating functions with filters"));
       return testDeleteWithFilter();
     })
-    .then(function () {
-      console.log(clc.green("\u2713 Test passed: deleting functions with filters"));
-      return testUnknownFilter();
```

Comment:
Why are we yanking this test? Also, we might want to make this use async so it's not so confusing that the console.log is for the previous function block.

Reply:
Yeah promise chain here is yucky, but it was written in the time when maybe it was in vogue? I'm not trying to refactor this test in any way - just want it to be not broken. If I'm actually making additions to this test, I will refactor but probably not worth it at this time.

Reply:
Also, copying snippet from the PR description for why I'm removing this:

> I'm removing the unknownFilter test because we now treat any filter as a valid one (Deploy code simply deploys nothing and reports "Deploy Complete"). This is subpar experience, but I've found fix to the issue to be a little more than few lines of change, and I'm not keen on fixing this bug right now.